### ISSUE TITLE
ip route get implemented ?

### ISSUE THREAD

**Issue by la-urre** [diosney/node-iproute#5]
Hello,

I can't find the equivalent for "ip route get " in your library, is it implemented ? Do you plan to implement it ?

How could I get the equivalent functionality ?

Thanks
Dam

**Comment by diosney** [comment]
Hi @laurre, thanks for using the library, I will take a look and let you know, give me a day or two to refresh my memory and gets my hands dirty :D

**Comment by laurre** [comment]
Cool, thanks @diosney !  I would be glad  to help if needed (and also as a tester for sure)

**Comment by diosney** [comment]
I counted in Neptune's days, not Earth ones, lol.

Anyways, I just published version `2.0.0-beta`, which is an overhaul of the library and fixes a lot of issues, this one included, since allowed me to add support for `ip route get`.

See [route.get](https://diosney.github.io/node-iproute/functions/route.get.html) for the full reference on it.

The beta tag is mainly to keep it a week or two for you to test it and let me know of anything before publishing the final version, but it should be fine.
